# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../enigma/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur enigma-0.92-orig/src/file_internal.hh enigma-0.92/src/file_internal.hh
--- enigma-0.92-orig/src/file_internal.hh	2004-10-22 17:26:25.000000000 +0000
+++ enigma-0.92/src/file_internal.hh	2006-04-24 10:47:58.454124250 +0000
@@ -19,7 +19,7 @@
 
     class GameFS {
     public:
-        GameFS::GameFS();
+        GameFS();
 
         void clear() { entries.clear(); }
 		
diff -Nur enigma-0.92-orig/src/stones_simple.cc enigma-0.92/src/stones_simple.cc
--- enigma-0.92-orig/src/stones_simple.cc	2005-03-02 20:33:36.000000000 +0000
+++ enigma-0.92/src/stones_simple.cc	2006-04-24 10:52:34.731390500 +0000
@@ -1672,7 +1672,7 @@
             KillStone(p);
             SetItem(p, it_explosion1);
         }
-        void BombStone::message(const string &msg, const Value &) {
+        void message(const string &msg, const Value &) {
             if (msg =="expl" || msg =="bombstone")
                 change_state(BREAK);
         }
